backend/package.json

```diff
@@ -28,6 +28,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "ajv": "^6.12.2",
```

Comment:
ajv is included in fastify already, i think it was previously removed because it was in there to override the configuration but it's not needed as a separate dependency (might cause conflicting versions as well?)

Reply:
The first time I tried to run the server node could not find ajv 🤷‍♂️